### PR TITLE
Introduce Auto-Format to Sprig Editor #1399 - Edited

### PR DESCRIPTION
**Pull request by DevIos01**
after the feedback from @leomcelroy on #1475 , i have changed the option as leo tested one which seems to be the way we want on editor.

### CODE REVIEW

**Review by JosiasAurel — APPROVED** [review]
lgtm

**Comment by leomcelroy** [comment]
This also opens all of the collapsed assets. You should probably collapse the assets by default when this is pressed.

**Comment by leomcelroy** [comment]
Collapse the assets means to close the text of sprites, maps, and tunes.

**Comment by DevIos01** [comment]
Ah okie, I will commit the fix soon :)

**Comment by grymmy** [comment]
lgtm, agree w/ feedback provided already

File: src/components/navbar-editor.tsx [hunk 1]
```diff
@@ -13,6 +13,8 @@ import { usePopupCloseClick } from '../lib/utils/popup-close-click'
 import { upload, uploadState } from '../lib/upload'
 import { VscLoading } from 'react-icons/vsc'
 import { defaultExampleCode } from '../lib/examples'
+import beautifier from "js-beautify";
+// import { js_beautify } from 'js-beautify';
```
Comment (grymmy) [inline src/components/navbar-editor.tsx:17]: nit: is it important to keep this?

**Review by grymmy — APPROVED** [review]

**Comment by DevIos01** [comment]
Hey @leomcelroy i have added the collapse after format lmk if this is done now :)

**Comment by leomcelroy** [comment]
Okay looks good to me.

**Pull request merged**